src/app/settings/user/user.component.html

```diff
@@ -1,35 +1,51 @@
 <ng-container *ngIf="(user$ | async) as user">
     <div class="container">
-        <h1 class="app-page-title">USER PROFILE</h1>
+        <h1 class="app-page-title">{{ 'settings.user.title' | translate }}</h1>
 
         <div>
-            <div class="row">
-                <div class="col-md-3">
-                    <div class="mt-4">
-                        <div class="explainer-mini">First name</div>
-                        {{ user.first_name }}
-                    </div>
+            <div class="row mt-3">
+                <div class="col-md-4">
+                    <label for="first_name" class="app-page-label">
+                        {{ 'settings.user.first_name' | translate }}
+                    </label>
+                    <input class="form-control p-4" id="first_name" disabled type="text"
+                           [value]="user.first_name">
```

Comment:
Nice! Btw. I don't know for you, but `input-lg` is awful as f**k. In design pages, it might look fine, but in reality, input fields are big as they are designed for elders. I'll use `input-reg` instead.

Reply:
Right, that makes sense. We should swap it through the app then and remove `input-lg` class so that we stay more consistent.

Reply:
ah, it sometimes looks OK when a primary button is in the same line. idk.